public/scripts/actions.jsx

```diff
@@ -624,3 +625,27 @@ export function hydrate() {
     dispatch(fetchSklearnModels());
   };
 }
+
+
+export function receivePredResultsCSV(data) {
+  return {
+    type: RECEIVE_PRED_RESULTS_CSV,
+    payload: data
+  };
+}
+
+
+export function downloadPredictionCSV(ID) {
```

Comment:
Is this level of indirection needed?  Why not just send the user straight to a URL serving CSV?